dynomitemanager/src/main/resources/dynomitemanager.properties

```diff
@@ -92,8 +92,14 @@ dm.dynomite.storage.preconnect=true
 # ENV: DM_DYNOMITE_MULTI_DC
 dm.dynomite.multi.dc=true
 
-#florida.dyno.mbuf.size=16384
-#florida.dyno.allocated.messages=200000
+# Dynomite memory buffer (mbuf) size in bytes
+# OLD: florida.dyno.mbuf.size=16384
+# ENV: DM_DYNOMITE_MBUF_SIZE
+dm.dynomite.mbuf.size=16384
```

Comment:
I have kept backward compatibility with the environment variables.

However, I have broken backward compatibility with the FP names, although this is only an issue when a non-default value is set.

If we need to maintain backward compatibility then I can update each configuration method to first check the old property, and if it's null, then use Archaius.

Let me know if this approach sounds okay. If we need to maintain backward compatibility with the FP keys, then I'll make this update to each method.